### DISCUSSION TITLE
Product bundling

### DISCUSSION THREAD

**Discussion by amirbalouchi** [saleor/saleor#10884] [General] [answered]
Hi everyone,

I need something like product bundling to group complementary products into a single unit and set a lower price than if they were sold individually. 
It seems there is no feature like this on the Saleor.
I'm willing to develop this feature for Saleor, but before I get to it, I would like to know if you have any suggestions/tips for me.

Thanks

**Comment by amirbalouchi** [comment]
@onecrayon @curaloucura @patrys @borekb  any suggestions?

**Reply by timuric** [reply]
Feel free to share details of your use case, it will be useful for us since we are about to start building this feature

**Comment by peelar** [comment, marked as answer]
Hello @amirbalouchi 👋 

Currently, there is no such feature in Saleor but we are working on conceptualizing it and you can expect to see RFC on it in the near future.

**Reply by PavelHonza** [reply]
Do you need help with developing this feature?

**Reply by amirbalouchi** [reply]
@PavelHonza 
Yes I do 
If we can contribute to developing this feature, it would be great.

**Comment by PavelHonza** [comment]
We decide to leave this project with our company. Because we miss the basic functions.E.g. Discount per customer, Customer groups and BoundlingOn 28 Dec 2023 06:42, amirbalouchi ***@***.***> wrote:
@PavelHonza
Yes I do
If we can contribute to developing this feature, it would be great.

**Reply by timuric** [reply]
Hey @PavelHonza 
Customer groups are on our roadmap, you can expect it in the near future. Bundles can be made using reference attributes.

**Comment by Qasem-h** [comment]
When will Product Bundles be available for use in Saleor?
